KsApi/models/Project.swift

```diff
@@ -324,6 +326,8 @@ extension Project: Decodable {
     self.urls = try values.decode(UrlsEnvelope.self, forKey: .urls)
     self.video = try values.decodeIfPresent(Video.self, forKey: .video)
     self.watchesCount = nil
+    self.isPledgeOverTimeAllowed =
+      try values.decodeIfPresent(Bool.self, forKey: .isPledgeOverTimeAllowed) ?? false
```

Comment:
nit: I haven't seen any spike docs for this yet, so checking: is the plan to add this field to the v1 API as well as to the graphql project? If not, I think we should consider making this an optional bool instead, and setting it to `nil` when the field is unavailable.

Reply:
The field will only be added to GraphQL for now, so I agree that using an optional bool would be a good approach.